lib/tram/policy/error.rb

```diff
@@ -99,10 +95,10 @@ def contain?(some_key = nil, **some_tags)
     UNDEFINED = Dry::Initializer::UNDEFINED
     DEFAULT_SCOPE = %w[tram-policy errors].freeze
 
-    def initialize(key, scope, **tags)
-      @key   = key
-      @tags  = tags
-      @scope = scope || DEFAULT_SCOPE
+    def initialize(key, **tags)
+      @key  = key
+      @tags = tags
+      @tags[:scope] ||= DEFAULT_SCOPE if key.is_a?(Symbol)
```

Comment:
hmmm. I'm what do you think about having nil scope as a valid case?
I'm not sure about it, but...
maybe we should prefer 
```suggestion
      @tags[:scope] = @tags.fetch(:scope) { DEFAULT_SCOPE } if key.is_a?(Symbol)
```

Reply:
I couldn't imagine the case when this is necessary. But why not? At least, your version is less magical for explicit `nil`